### PR TITLE
[#31039] Add tab styling

### PR DESCRIPTION
**Pull request by bienvu**
Added tab style.
Please review!

### CODE REVIEW

File: themes/agov/agov_whitlam/sass/components/tabs/_tabs.scss [hunk 1]
```diff
@@ -18,8 +18,10 @@
 .tabs,
 %tabs {
   @extend %clearfix;
-  @include background-image(linear-gradient(bottom, color(tabs-border) 1px, transparent 1px));
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/tabs/_tabs.scss]: If you bring this back, the 2px grey line below the tabs can be hidden by the active 2px blue border on the active tab.

File: themes/agov/agov_whitlam/sass/components/tabs/tabs--secondary.hbs [hunk 2]
```diff
@@ -2,4 +2,4 @@
   <li class="tabs--secondary__tab is-active"><a href="#" class="tabs--secondary__tab-link is-active">Content <span class="visually-hidden">(active tab)</span></a></li>
   <li class="tabs--secondary__tab"><a href="#" class="tabs--secondary__tab-link">Display</a></li>
   <li class="tabs--secondary__tab"><a href="#" class="tabs--secondary__tab-link">Settings</a></li>
-</ul>
+</ul>
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/tabs/tabs--secondary.hbs]: Revert this change, please.

File: themes/agov/agov_whitlam/sass/components/tabs/tabs.hbs [hunk 3]
```diff
@@ -3,3 +3,8 @@
   <li class="tabs__tab"><a href="#" class="tabs__tab-link">Edit</a></li>
   <li class="tabs__tab"><a href="#" class="tabs__tab-link">Revisions</a></li>
 </ul>
+<div class="tabs-content">
+  <div class="tabs-content__default">
+   Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book
+  </div>
+</div>
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/tabs/tabs.hbs]: Why do we need `tabs-content` and `tabs-content__default` elements?

**Comment by bienvu** [comment]
sorry. I just added to make spacing between tabs and content.

**Comment by joannguyen** [comment]
test this please
